src/AppBundle/Controller/SecurityController.php

```diff
@@ -25,8 +25,7 @@
 class SecurityController extends Controller
 {
     /**
-     * @Route("/login", name="security_login_form")
-     * @Method("GET")
```

Comment:
This had to be removed, as the router will error when we submit the form. It wasn't very usefull anyways...